tox.ini

```
@@ -74,8 +74,16 @@ commands =
 deps =
   {[testenv]deps}
   -r{toxinidir}/docs/requirements.in
+  # direct dependencies
   pylint
   pylint-pytest < 1.1.0
+  # indirect dependencies
```

That block separator I asked for:
```suggestion

  # indirect dependencies
```